cbindgen.toml

```
@@ -2,6 +2,11 @@ language = "C"
 tab_width = 2
 include_guard = "DDTRACE_PHP_H"
 style = "both"
+header = """// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+struct _zend_string;
```

```suggestion
header = """struct _zend_string;
```
We have the LICENSE file (which by the way dual-licenses BSD and apache2). Don't add an arbitrary license here.